docs/book/src/topics/dual-stack.md

```diff
@@ -0,0 +1,67 @@
+# Dual-stack clusters
+
+## Overview
+
+CAPZ enables you to create [dual-stack](https://github.com/Azure/aks-engine/tree/master/examples/dualstack) Kubernetes cluster on Microsoft Azure.
```

Comment:
why are we pointing to AKS Engine docs here? should we point to official k8s docs instead?